worldedit-fabric/src/main/java/com/sk89q/worldedit/fabric/FabricWorld.java

```diff
@@ -170,7 +170,13 @@ public Path getStoragePath() {
         return null;
     }
 
-    public void markAndNotifyBlock(World world, BlockPos pos, @Nullable WorldChunk worldChunk, net.minecraft.block.BlockState blockState,
+    /**
+     * This is a heavily modified function stripped from MC to apply worldedit-modifications.
+     *
+     * It is taken from Forge's World class, not Fabric's. This code should be consistent with the
+     * ForgeWorld code.
+     */
+    private void markAndNotifyBlock(World world, BlockPos pos, WorldChunk worldChunk, net.minecraft.block.BlockState blockState,
```

Comment:
can we rename either `blockstate` or `state`? or is this just supposed to mirror Forge's naming as closely as possible for copy-ability.
If that's the case, maybe just a comment like `// blockState = existing, state = new, blockState2 = ... hold up that's the existing one`

Reply:
I've actually already corrected this while refactoring to allow Spigot to be more easily implemented.